core-domain/src/main/java/org/uniprot/core/uniparc/UniParcEntryLight.java

```diff
@@ -0,0 +1,36 @@
+package org.uniprot.core.uniparc;
+
+import org.uniprot.core.Sequence;
+import org.uniprot.core.util.Pair;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+import java.util.List;
+
+public interface UniParcEntryLight extends Serializable {
+    UniParcId getUniParcId();
```

Comment:
I also wonder whether separating the value and validation logic is possible.

Reply:
Which validation logic are you talking about?